compiler/rustc_middle/src/query/mod.rs

```diff
@@ -1009,6 +1013,14 @@ rustc_queries! {
         cache_on_disk_if { true }
     }
 
+    /// A list of types where the ADT requires drop if and only if any of
+    /// those types require drop. If the ADT is known to always need drop
+    /// then `Err(AlwaysRequiresDrop)` is returned. TODO: Change
+    query adt_insignificant_drop_tys(def_id: DefId) -> Result<&'tcx ty::List<Ty<'tcx>>, AlwaysRequiresDrop> {
```

Comment:
Doc needs to be updated, not sure what to put here

Reply:
One thing to ask niko would be do we need cache_on_disk, I'd say no. Keep it true for now

Reply:
adt_insignificant_drop_tys should this not be adt_significant_...